documentation/public/user-guide/08-examples/03-jump-to-definition/examples/jump-to-definition.mts

```diff
@@ -0,0 +1,50 @@
+import assert from "node:assert";
+import { TerminalKindExtensions } from "@nomicfoundation/slang/cst";
+import { CompilationUnit } from "@nomicfoundation/slang/compilation";
+import { findTerminalNodeAt } from "../../common/find-terminal-node-at.mjs";
+
+type Target = {
+  file: string;
+  line: number;
+  column: number;
+};
+
+export function jumpToDefinition(unit: CompilationUnit, fileId: string, line: number, column: number): Target | string {
```

Comment:
Here+others, I suggest using `Error` instead of `string`:

```suggestion
export function jumpToDefinition(unit: CompilationUnit, fileId: string, line: number, column: number): Target | Error {
```